parity/run.rs

```diff
@@ -314,6 +329,31 @@ pub fn execute(cmd: RunCmd, can_restart: bool, logger: Arc<RotatingLogger>) -> R
 	let client = service.client();
 	let snapshot_service = service.snapshot_service();
 
+	// initialize the local node information store.
+	// TODO: tick it every once in a while?
```

Comment:
in the case of a panic or other dirty exit where destructors aren't run, the list of transactions in the database won't be updated. we might want to tick periodically (once every 5 minutes?) to ensure a higher chance of them making it in.